File: modulo1/segmento3/apunte8EstrDeDatos/Ejercicio2.java
import java.util.Scanner;
import java.util.ArrayList;

public class Ejercicio2{
    public static void main(String args[]){
        Scanner teclado = new Scanner(System.in);
        ArrayList<Integer> miListaImpar = new ArrayList<Integer>();

        for(Integer i =0 ; i<=100;i++){
            if(i%2 !=0){
                miListaImpar.add(i);
            }
        }

        for(Integer i=0; i<miListaImpar.size()-1;i++){
            for(Integer k=i+1;k<miListaImpar.size();k++){
                if(miListaImpar.get(i)>miListaImpar.get(k)){
                    Integer auxiliar = miListaImpar.get(k);
                    miListaImpar.set(k,miListaImpar.get(i));
                    miListaImpar.set(i,auxiliar);
                }
        
            }
        }
        System.out.print(miListaImpar);

    }
}

File: modulo1/segmento3/apunte8EstrDeDatos/ejercicio2.py
miListaImpar=[]

for i in range(100):
    if i%2 !=0:
        miListaImpar.append(i)

for i in  range(len(miListaImpar)):
    for k in range(i,len(miListaImpar)):
            if miListaImpar[i]<miListaImpar[k]:
                aux = miListaImpar[k]
                miListaImpar[k]=miListaImpar[i]
                miListaImpar[i]=aux

print(miListaImpar)